algebra/src/curves/models/twisted_edwards_extended/tests.rs

```diff
@@ -33,27 +28,25 @@ pub fn edwards_curve_serialization_test<P: TEModelParameters>(buf_size: usize) {
         let a = a.into_affine();
         {
             let mut serialized = vec![0; buf_size];
-            a.serialize(&[], &mut serialized).unwrap();
+            a.serialize(&mut &mut serialized[..]).unwrap();
```

Comment:
```suggestion
            a.serialize(&mut serialized[..]).unwrap();
```

Reply:
FYI: the second `&mut` is required here as it needs to get a mutable pointer to a mutable slice here.
However, I agree that it is very hard to read. For better readability, I replaced it by using a cursor.
```
let mut cursor = Cursor::new(&mut serialized[..]);
a.serialize(&mut cursor).unwrap();
```